src/views/InitialLoad.vue

```diff
@@ -102,14 +115,20 @@ export default defineComponent({
         "systemJobEnumId_op": "in"
       }
     })
+    this.store.dispatch('webhook/fetchWebhooks')
   },
   computed: {
     ...mapGetters({
       getJobStatus: 'job/getJobStatus',
       getJob: 'job/getJob',
       shopifyConfigId: 'user/getShopifyConfigId',
-      currentEComStore: 'user/getCurrentEComStore'
-    })
+      currentEComStore: 'user/getCurrentEComStore',
+      getCachedWebhook: 'webhook/getCachedWebhook'
+    }),
+    fileStatusUpdateWebhook(): boolean {
+      const webhookTopic = this.webhookEnums['BULK_OPERATIONS_FINISH']
+      return this.getCachedWebhook[webhookTopic]?.topic === webhookTopic
```

Comment:
If we have topic as key, I think we could skip the comparison

Reply:
Done Sir